src/typescript.js

```diff
@@ -1,14 +1,17 @@
 
 const { Module } = require('module');
+
 const m = new Module('', null);
+const log = (...args) => { if (!quiet) { console.log(...args); } }
 m.paths = Module._nodeModulePaths(process.env.TYPESCRIPT_LOOKUP_PATH || (process.cwd() + '/'));
 let typescript;
 try {
   typescript = m.require('typescript');
-  console.log("ncc: Using typescript@" + typescript.version + " (local user-provided)");
+  if (!global.nccQuiet) console.log("ncc: Using typescript@" + typescript.version + " (local user-provided)");
```

Comment:
Can this be solved without a global variable?

Reply:
another solution I can think of is: 
create a config file for CLI arguments/flags, can update the config when args parsed the arguments from command line. other files will use this config file to consume flags.

```
// index.js
import cliargs from './cliargs'

params = args.parse()
cliargs.update(params)

// typescript.js
import cliargs from './cliargs'

quite = cliargs.get('quiet')
```

if you feel this is proper I'll update the PR later